src/components/Heading.jsx

```diff
@@ -1,5 +1,7 @@
+import Image from "next/image";
+import photo from "public/colored-graph.svg";
 const Heading = () => {
-  return <div>Heading</div>;
+  return <Image src={photo} />;
```

Comment:
can you pass in the heading text as props and have it display next to the image. also include the little yellow bar below the text.

check out the image in the issue for reference